physics/src/modules/AtmosphereBoundaryModule/ERA5Atmosphere.cpp

```diff
@@ -49,6 +54,20 @@ void ERA5Atmosphere::configure()
     Finalizer::registerUnique(Module::finalize<IFluxCalculation>);
 
     filePath = Configured::getConfiguration(keyMap.at(FILEPATH_KEY), std::string());
+    doChecks = Configured::getConfiguration(keyMap.at(CHECKS_KEY), false);
+
+    if (doChecks) {
+        // clang-format off
+        fieldsToCheck.push_back({pfx+": tair",  &tair,  {  -100, 80}});
+        fieldsToCheck.push_back({pfx+": tdew",  &tdew,  {  -100, 80}});
+        fieldsToCheck.push_back({pfx+": pair",  &pair,  { 700e2, 1100e2}});
+        fieldsToCheck.push_back({pfx+": sw_in", &sw_in, {    -1, 2e3}});
+        fieldsToCheck.push_back({pfx+": lw_in", &lw_in, {     0, 1e3}});
+        fieldsToCheck.push_back({pfx+": wind",  &wind,  {  -0.1, 150}});
+        fieldsToCheck.push_back({pfx+": uwind", &uwind, {  -150, 150}});
+        fieldsToCheck.push_back({pfx+": vwind", &vwind, {  -150, 150}});
```

Comment:
I think this would look neater if you merged in a initializer list literal. There would at least be only one function call. But this one is a matter of taste.